src/routes/documentation.js

```diff
@@ -18,9 +18,10 @@ const jsonMiddleware = (req, res, next) => {
 };
 
 router.get("/info", (req, res) => {
+  const iconLink = `${process.env.HOST}/icon.svg`;
   const { title, description } = openApiDocumentation.info;
```

Comment:
I guess reading `HOST` from ```.env``` will be same as is in our remote repos

Reply:
What remote repos?

Reply:
Never mind bro..I just checked the config vars on heroku and it has the live staging server set to the heroku server